Pod/Classes/PickerView.swift

```diff
@@ -41,7 +41,7 @@ public class PickerView: UIView {
     - parameter Infinite: Loop through the data source offering a infinite scrolling experience to the user.
     */
     
-    public enum ScrollingStyle: Int {
+    public enum ScrollingStyle {
```

Comment:
@bguidolim removing the associated value from `ScrollingStyle` causes a crash because we are instantiating the member values through the rawValues according to values provided by UISegmentedControl selected indexes of the `Example` app. (Yes, I know, it seems to don't make any sense, but I'll implement @IBInspectable to this enum soon, so we will need the associated values anyway)

This comment is valid to the `SelectionStyle` enum too.